README.md

```diff
@@ -25,14 +25,18 @@ and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/vers
 1. Fork and clone the Git repo:
    `git clone https://github.com/<your_username>/python-zulip-api.git`
 
-2. Make sure you have [pip](https://pip.pypa.io/en/stable/installing/)
+2. Setup upstream and fetch upstream/master (Required for Gitlint):
+   - `git remote add upstream https://github.com/zulip/python-zulip-api.git`
+   - `git fetch upstream master`
```

Comment:
Maybe worth just including these in step 1 in a single 3-line code block so folks can copy-paste?  Though I think if you test that you'll notice we need a `cd` command too, which is well worth adding.